src/gam/__init__.py

```diff
@@ -3568,6 +3637,14 @@ def getDriveFileAttribute(i, body, parameters, myarg, update=False):
     return i
 
 
+def check_num_parents(user, body):
```

Comment:
function name should match what we are returning so we can do something like:

```
if has_multiple_parents(body):
    # warn user
```
and we get an idea of what has_multiple_parents does and returns just by it's name.

Also, let's move the stderr message out of the function and just make the function return true/false. We can output something within the if statement if needed.